extensions/wrapper/wrapper-common-mappers/build.gradle.kts

```diff
@@ -15,10 +15,16 @@ dependencies {
     compileOnly("org.projectlombok:lombok:${lombokVersion}")
 
     api("${edcGroup}:policy-model:${edcVersion}")
+    api("${edcGroup}:core-spi:${edcVersion}")
+    api("${edcGroup}:transform-core:${edcVersion}")
+
+
     api(project(":extensions:wrapper:wrapper-common-api"))
 
     implementation("org.apache.commons:commons-lang3:3.13.0")
     implementation("org.apache.commons:commons-collections4:4.4")
+    implementation("cz.cvut.kbss.jsonld:jb4jsonld-jackson:0.13.2")
```

Comment:
Are you using this library? Does this library work well? I mean the example looks alright: https://github.com/kbss-cvut/jb4jsonld-jackson/blob/master/example/src/main/java/cz/cvut/kbss/jsonld/jackson/example/model/Organization.java

Although the library has basically 9 stars only or sth